src/build/env.py

```diff
@@ -187,6 +188,25 @@ def _create_isolated_env_virtualenv(path):  # type: (str) -> Tuple[str, str]
 # venv only exists on Python 3+
 if sys.version_info >= (3,):  # noqa: C901
 
+    once = functools.lru_cache(maxsize=None)
+
+    @once
+    def _fs_supports_symlink():  # type: () -> bool
+        """Return True if symlinks are supported"""
+        # Using definition used by venv.main()
+        if os.name != 'nt':
+            return True
+
+        # Windows may support symlinks (setting in Windows 10)
+        with tempfile.NamedTemporaryFile(prefix='TmP') as tmp_file:
+            temp_dir = os.path.dirname(tmp_file.name)
+            dest = os.path.join(temp_dir, '{}-{}'.format(tmp_file.name, 'b'))
```

Comment:
Why do we need to do all this instead of just suffiixng "-b" to the filename?

Reply:
Umm.... Because that's what was done before and I inherited it? I see no reason why we can't just do that, yes.

Reply:
Hadn't noticed it was inherited :)